grpc-ballerina/tests/20_unary_client_for_anonymous_service_client.bal

```diff
@@ -50,6 +50,46 @@ function testAnonymousUnregisteredService() returns error? {
         test:assertEquals(err.message(), "Error when initializing service register builder. Invalid service path. " + 
             "Service path cannot be nil");
     } else {
-        test:assertFail("Expecter internal error not found");
+        test:assertFail("Expected internal error not found");
+    }
+}
+
+@test:Config {dependsOn: [testAnonymousUnregisteredService], enable: true}
+function testAnonymousServiceWithoutRPCImplemented() returns error? {
+    string msg1 = "Error when initializing service register builder. " + 
```

Comment:
```suggestion
    string msg1 = "Error when registering the service. " + 
```

Reply:
shall we say like, `Error while registering the service`. register builder is internal detail

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/314/commits/66bd794f1577a9bf4882fe80526857bfed367e02